t/ui/16-activity-view.t

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env perl
+# Copyright (C) 2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+use Test::Most;
+
+use Test::Mojo;
+use Test::Warnings ':report_warnings';
+
+use FindBin;
+use lib "$FindBin::Bin/../lib";
+use OpenQA::Test::TimeLimit '20';
+use OpenQA::Test::Case;
+use OpenQA::Client;
+
+use OpenQA::SeleniumTest;
+
+OpenQA::Test::Case->new->init_data(fixtures_glob => '01-jobs.pl');
+
+plan skip_all => $OpenQA::SeleniumTest::drivermissing unless my $driver = call_driver;
+
+my $t = Test::Mojo->new('OpenQA::WebAPI');
+# we need to talk to the phantom instance or else we're using the wrong database
+my $url = 'http://localhost:' . OpenQA::SeleniumTest::get_mojoport;
+
+subtest 'Login required' => sub {
+    $driver->find_element_by_link_text('Login')->click();
+    is($driver->find_element('#user-action a')->get_text(), 'Logged in as Demo', 'logged in as demo');
+    $driver->get('/admin/activity_view');
```

Comment:
the subtest is called "Login required" but this test does not actually check this. Maybe first check that without login directly accessing the deep URL would render some error?

Reply:
I can't find any existing Selenium tests for such cases so I guess I'll drop it for now and rely on the definition of the route in `WebAPI.pm` which ensures that.

Reply:
yes, please drop it